src/generators/java/renderers/EnumRenderer.ts

```diff
@@ -49,7 +49,7 @@ ${this.indent(this.renderBlock(content, 2))}
       break;
     }
     default: {
-      key = String(value);
+      key = FormatHelpers.replaceSpecialCharacters(String(value), { exclude: [' '], separator: '_' }); 
       //Ensure no special char can be the beginning letter 
       if (!(/^[a-zA-Z]+$/).test(key.charAt(0))) {
         key = `string_${key}`;
```

Comment:
You can remove the old special chars code 🙂

Reply:
I think this is still needed since even with the new escape function it can happen that the enum key starts with a number and that would lead to compile errors. 

The same code is also still present in the typescript `EnumRenderer`
https://github.com/asyncapi/modelina/blob/2a8659da465f53f9cfe1097a622ed82d0fa7594c/src/generators/typescript/renderers/EnumRenderer.ts#L55

Maybe I can move the check to a `FormatHelpers` function to remove duplication?

Reply:
You are right, good catch!

If you see it as being beneficial and it removes duplicated code, feel free to do so! 🙂